### DISCUSSION TITLE
UnInstall ImageMagick

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#2712] [Help]
How to uninstall ImageMagick completely in Ubuntu

**Comment by urban-warrior** [comment]
Was it installed as a package?  As an RPM?  Or from source?  The answer is different depending on the install method.

**Reply by juanjcardona13** [reply]
Maybe you could give an example of each one, please
For future searches or knowing where you are documented
Thank you
